src/main/java/ua/softserveinc/tc/dto/UserAbonnementDto.java

```diff
@@ -1,18 +1,18 @@
 package ua.softserveinc.tc.dto;
 
 public class UserAbonnementDto {
-    Long userId;
+    Long[] userId;
```

Comment:
List isbetter. For example : you want to add one more abonnement to the DTO -  array have fixed length and it will dtop an error